content/blog/wingingit/winging-it-11.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Web Components
+episode: 11
+sponsors: false
+tags:
+  - Web Components
+author:
+  - miriam
+  - james
+date: 2024-08-27
+length: 60 mins
+image:
+  src: winging-it/winging-it-11.jpg
+media:
+  - youtube: c9l-01cySkQ
+summary: |
+  Web Components are a polarizing feature
+  that seem simultaneously old news
+  and not quite ready for production yet.
+  But we’ve been making things with Web Components,
+  and finding some areas where they really work well.
+  Join us,
+  along with special guest [Zach Leatherman](https://www.zachleat.com/)
+  of [11ty](https://www.11ty.dev/),
+  to hear more about how we’re using them,
+  what they work well for,
+  and why we’re excited about them.
+---
+
+{% import "embed.macros.njk" as embed %}
+{% import "utility.macros.njk" as utility %}
+
+{{ embed.figure(
+  data=media
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## What we cover:
+- When did Web Components “click”
+- How to incorporate progressive enhancement
+- Future Web Components projects
+- What is still missing from Web Components?
+- Thoughts on lit
+
+## Demos:
+- [Slide-deck](https://github.com/oddbird/slide-deck/)
+
+- [Ground control](https://github.com/mirisuzanne/ground-control)
+
+- [is-land](https://github.com/11ty/is-land)
+
+- [fundraising-status](https://github.com/zachleat/fundraising-status)
+
+- [An Attempted Taxonomy of Web Components](https://www.zachleat.com/web/a-taxonomy-of-web-component-types/)
+
+- [Progressive Enhancement Recipes using Eleventy WebC Image Comparison Components](https://demo-webc-image-compare.netlify.app/)
```

Comment:
```suggestion
- [Slide-deck](https://github.com/oddbird/slide-deck/)
- [Ground control](https://github.com/mirisuzanne/ground-control)
- [is-land](https://github.com/11ty/is-land)
- [fundraising-status](https://github.com/zachleat/fundraising-status)
- [An Attempted Taxonomy of Web Components](https://www.zachleat.com/web/a-taxonomy-of-web-component-types/)
- [Progressive Enhancement Recipes using Eleventy WebC Image Comparison Components](https://demo-webc-image-compare.netlify.app/)
```

Reply:
Also, I don't think I've seen slide-deck and ground-control with any capital letters. Not sure if lower case would make sense since is-land is also lower?